src/utils/dex/encounters.js

```diff
@@ -65,8 +66,8 @@ function getAllIncenseEncounters(areaEncounters) {
   return {}
 };
 
-function getTODEncounters(areaEncounters) {
-  const filteredEncounters = areaEncounters.filter(obj => TOD_ENC.includes(obj.encounterType));
+function getTimeOfDayEncounters(areaEncounters) {
```

Comment:
Looks like this function returns an array normally, or an empty object if it's nothing. It would be easier to handle empty responses if it returned an empty array instead, imo. Up for debate.

Reply:
Agreed